doc/articles/toc.yml

```diff
@@ -110,6 +110,11 @@
               href: guides/silverlight-migration/98-timeentry-samples.md
             - name: Useful resources
               href: guides/silverlight-migration/99-useful-resources.md
+        - name: WPF Migration
```

Comment:
Not really the right section, since the content is not a step-by-step tutorial. You can put it under Development > Common Tasks, alongside 'Migrating UWP-only code' etc.

Reply:
@davidjohnoliver 

@sasakrsmanovic mentioned in chat that I should put it at the same level as the Silverlight migration docs, but your idea sounds better.